examples/multilang-multi-project/proto/lib/google/protobuf/descriptor.proto

```
@@ -0,0 +1,1337 @@
+option csharp_namespace = "Google.Protobuf.Reflection";
+option go_package = "google.golang.org/protobuf/types/descriptorpb";
+option java_outer_classname = "DescriptorProtos";
+option java_package = "com.google.protobuf";
+option objc_class_prefix = "GPB";
+// descriptor.proto must be optimized for speed because reflection-based
+// algorithms don't work during bootstrapping.
+option optimize_for = SPEED;
+
+// The protocol compiler can output a FileDescriptorSet containing the .proto
+// files it parses.
+message FileDescriptorSet {
```

Buf (1.54.0)

symbol "google.protobuf.FileDescriptorSet" already defined at google/protobuf/descriptor.proto:56:9

(COMPILE)